tableaudocumentapi/xml_open.py

```diff
@@ -0,0 +1,28 @@
+try:
+    from distutils2.version import NormalizedVersion as Version
```

Comment:
This feels heavy to me for what we're doing.

The strings are ordered and can be compared ("8.2" < "9.0 == True), can we just do
```
MIN_SUPPORTED_VERSION = "9.0"
...
 file_version = tree.getroot().attrib.get('version', '0.0')
    print(file_version, MIN_SUPPORTED_VERSION) # And remove this print, it's just debugging?
    if file_version < MIN_SUPPORTED_VERSION:
        raise VersionNotSupportedException(file_version)
    return tree
```